tradeexecutor/analysis/trade_analyser.py

```diff
@@ -196,6 +204,9 @@ def __post_init__(self):
         self.winning_take_profits_percent = calculate_percentage(self.winning_take_profits, self.take_profits)
         self.losing_take_profits_percent = calculate_percentage(self.losing_take_profits, self.take_profits)
 
+        days_as_decimal = self.duration.total_seconds() / (60 * 60 * 24)
+        self.average_position_frequency = self.total_positions / days_as_decimal if days_as_decimal else 0
+
     def get_trading_core_metrics(self) -> Dict[str, str]:
```

Comment:
`get_trading_core_metrics()` is displayed in the tooltip, so let's not cram too much here